Lesson27/ezgi/create/create-script.js

```diff
@@ -0,0 +1,46 @@
+// const urlParams = new URLSearchParams(window.location.search);
+// const postId = urlParams.get('postId');
+
+document.addEventListener('DOMContentLoaded', function () {
+
+const form= document.getElementById('create-post');
+form.addEventListener('submit', function(event){
+    event.preventDefault();
+
+    const newTitle=document.getElementById('title').value;
+    console.log(newTitle);
+    const newBody=document.getElementById('body').value;
+    console.log(newBody);
+
+    const newPost = {
```

Comment:
You can remove this object as it's not used anywhere except console.log statement below.